test/boost/README.md

```diff
@@ -68,6 +68,48 @@ Boost tests can also be run using test.py - which is a script that provides
 a uniform way to run all tests in scylladb.git - C++ tests, Python tests,
 etc.
 
+## Execution with pytest
+
+To run all tests with pytest execute 
+```bash
+pytest test/boost
+```
+
+To execute all tests in one file, provide the path to the source filename as a parameter
+```bash
+pytest test/boost/aggregate_fcts_test.cc
+```
+Since it's a normal path, autocompletion works in the terminal out of the box.
+
+To execute only one test function, provide the path to the source file and function name
+```bash
+pytest --mode dev test/boost/aggregate_fcts_test.cc::test_aggregate_avg
+```
+
+To provide a specific mode, use the next parameter `--mode dev`,
+if parameter isn't provided pytest tries to use `ninja mode_list` to find out the compiled modes.
```

Comment:
A thought: (I didn't go back to verify)
Please make sure that "ninja mode_list" is NOT run at all when it's not needed - especially when running test/cqlpy or test/alternator. It takes a non-negligible amount of time - about 0.7 seconds on my machine (this is https://github.com/scylladb/scylladb/issues/17361 but unfortunately Avi closed it)

Reply:
As for now, it only used in boost/unit test to find the available modes when they are not provided. Later on, it will be used for all tests to find out what binary to use.

Reply:
So to clarify, if I do "cd test/alternator; pytest" or test/alternator/run, this "ninja" isn't run?

I don't mind if you have a fixture that finds these modes and tests that need it will use that fixture. What I'm scared is that some auto-use fixture (like that "report plugin" thing) will cause it to be calculated even for tests that don't need it.

Reply:
Correct.
But later this fixture will be in alternator as well when there will be no mode provided.